libkbfs/node.go

```diff
@@ -34,6 +37,13 @@ func (c *nodeCore) ParentID() NodeID {
 	return c.parent.GetID()
 }
 
+// String is to support printing *nodeCore as a NodeID. If we want to
+// print nodeCores as nodeCores (e.g., for debugging) we might have to
+// implement Formatter.
+func (c *nodeCore) String() string {
+	return fmt.Sprintf("%p", c)
```

Comment:
Apparently, doing printf "%p" on an interface type is not liked by go vet, but since we use our own functions it wasn't caught.